.github/workflows/build.yml

```
@@ -0,0 +1,32 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-java-with-maven
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Setup Maven Action
+      uses: s4u/setup-maven-action@v1.7.0
+      with:
+       checkout-fetch-depth: 0
+       java-version: 17
+       java-distribution: temurin
+       maven-version: 3.9.6
+    - name: Build with Maven
+      run: mvn --debug -Dsurefire.failIfNoSpecifiedTests=false -Dtest=\!MissingWebsocketDependenciesIT* -B clean install
```

```suggestion
      run: mvn --debug -Dsurefire.failIfNoSpecifiedTests=false -Dtest=\!MissingWebsocketDependenciesIT* -B -f http clean install
```
specify http as the file to build will not require to use a root pom ....

---

Ah right, like that, yes, that could also work. Then we could drop the root pom indeed.